src/screens/FeedScreen/Content.js

```diff
@@ -340,7 +353,8 @@ Content.propTypes = {
   topics: PropTypes.arrayOf(PropTypes.string),
   item: PropTypes.object,
   onNewPollFetched: PropTypes.func,
-  setHaveSeeMore: PropTypes.func
+  setHaveSeeMore: PropTypes.func,
+  setIsShortText: PropTypes.func
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The prop `setIsShortText` has been added to PropTypes validation. This is good for ensuring that the correct type of prop is passed to the component.
```diff
+ setIsShortText: PropTypes.func
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->